README.md

```diff
@@ -2,6 +2,27 @@
 
 # Distributed Linear Algebra from the Future
 
+DLA-Future is a distributed linear algebra library implemented using using C++ `std::execution` [P2300](https://cplusplus.github.io/sender-receiver/execution.html) which provides:
+- An asynchronous C++ interface,
+- A synchronous C interface,
+- A synchronous ScaLAPACK-like C interface (ScaLAPACK drop-in with a subset of ScaLAPACK arguments. E.g. workspace arguments are not present),
+- A synchronous ScaLAPACK-like Fortran interface (See [DLA-Future-Fortran](https://github.com/eth-cscs/DLA-Future-Fortran)).
```

Comment:
```suggestion
- An asynchronous C++ interface,
- a synchronous C interface,
- a synchronous ScaLAPACK-like C interface (ScaLAPACK drop-in with a subset of ScaLAPACK arguments. E.g. workspace arguments are not present),
- a synchronous ScaLAPACK-like Fortran interface (See [DLA-Future-Fortran](https://github.com/eth-cscs/DLA-Future-Fortran)).
```
? Since we are using punctuation in the bullet list.